src/e2e-test/java/io/cdap/plugin/utils/GCConstants.java

```diff
@@ -0,0 +1,22 @@
+package io.cdap.plugin.utils;
+
+/**
+ * constants.
+ */
+public class GCConstants {
```

Comment:
What is GCConstants? rename to something more meaningful?

Reply:
Renamed.